### DISCUSSION TITLE
Running all tasks in a view

### DISCUSSION THREAD

**Discussion by sandygmaharaj** [semaphoreui/semaphore#1922] [Q&A] [answered]
Hi
I have created multiple tasks in a view and would like to run them all with a single click, is it possible?

**Comment by fiftin** [comment, marked as answer]
No, it is not possible. But it is great idea.

**Reply by sandygmaharaj** [reply]
How can I move this to a desired feature?

**Reply by fiftin** [reply]
I already created feature request: https://github.com/semaphoreui/semaphore/issues/1935

**Reply by btzq** [reply]
Hi Guys, i have the same feature request too! 

But i dont think its picked up as a feature to be built yet... i cant find any ticket about it.
